cmd/infra/azure/destroy.go

```diff
@@ -64,29 +63,27 @@ func (o *DestroyInfraOptions) Run(ctx context.Context) error {
 			return fmt.Errorf("failed to read the credentials: %w", err)
 		}
 	}
-	authorizer, err := auth.ClientCredentialsConfig{
-		TenantID:     creds.TenantID,
-		ClientID:     creds.ClientID,
-		ClientSecret: creds.ClientSecret,
-		AADEndpoint:  azure.PublicCloud.ActiveDirectoryEndpoint,
-		Resource:     azure.PublicCloud.ResourceManagerEndpoint,
-	}.Authorizer()
+
+	_ = os.Setenv("AZURE_TENANT_ID", creds.TenantID)
```

Comment:
why does this need to be set in the env now?

Reply:
From my understanding, this is how we would authenticate with Azure. It has a process of authenticating a user to get valid Azure credentials for creating/destroying resources.

https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity#readme-key-concepts
https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity#DefaultAzureCredential